hivemind/optim/grad_averager.py

```diff
@@ -11,6 +11,10 @@
 logger = get_logger(__name__)
 
 
+TGradientAverager = TypeVar("TGradientAverager", bound="GradientAverager")
```

Comment:
Question to Max: what's our naming convention for this? personally I'm not sure and fine with either option, but this looks like something that was standardized by more cultured devs :)